Readme.md

```diff
@@ -45,41 +45,50 @@ Prior to using Explorer for Endevor, ensure that you meet the following prerequi
 
 Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check [here](https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/) for more information.
 
-
 ## **User Guide**
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
 
 ### **Create a Profile**
 
-Create a Profile in Explorer for Endevor to specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
+Create a profile in Explorer for Endevor, then specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
 
 **Follow these steps:**
 
 1. Click on the + icon.
-2. Enter a name for your Profile.
+2. Enter a name for your profile.
 3. Enter your Endevor URL in the format `https://host:port`.
 4. (Optional) To add your mainframe credentials to your profile, enter your username and password. Adding your credentials to your profile lets you access different data sources without entering your credentials.
-5. Set profile settings to True or False to indicate that your profile should Reject or Accept connections with self-signed certificates.  
-Your Profile is now available in the panel on the left.
+5. Set your preference for whether Explorer for Endevor should Reject or Accept connections with self-signed certificates.
+    - **True**: Reject connections with self-signed certificates.
+    - **False**: Accept connections with self-signed certificates.
+    
+Your profile is now available in the panel on the left.
+
+### **Working with Data Sources**
+
+Now you have created your profile, assign the data sources that you want to work with.
```

Comment:
Now that you have created your profile, assign the configuration(s) that you want to work with.

Reply:
Done